### DISCUSSION TITLE
What 🏝️ is the setting for the novel Captain Corelli's Mandolin?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1096] [Galaxy Trivia] [answered]
**Bonus:** How many of the Bronte sisters were famous for their writing?

**Comment by peterbaccaro** [comment]
Captain Corelli's Mandolin was set on the **Greek island of Cephalonia** during the Italian and German occupation of the Second World War.

The **three Brontë sisters** Charlotte, Emily and Anne Brontë were the authors of some of the best-loved books in the English language.

**Comment by thuongtruong1009** [comment, marked as answer]
The setting for the novel "Captain Corelli's Mandolin" is the Greek island of Cephallonia (also spelled Kefalonia) during World War II.

There were three famous Brontë sisters known for their writing: Charlotte Brontë, Emily Brontë, and Anne Brontë. They were all English authors from the 19th century and are best known for their classic novels like "Jane Eyre" (Charlotte Brontë), "Wuthering Heights" (Emily Brontë), and "The Tenant of Wildfell Hall" (Anne Brontë).
